src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -269,36 +272,24 @@ private boolean isBlacklistProject(String project) {
         return ArrayUtils.contains(operationsProjects, project.toLowerCase());
     }
 
-    private boolean isOperationsUser(final String username) {
-        ConfigBuilder builder = new ConfigBuilder();
-
-        boolean clusterReaderAllowed = settings.getAsBoolean(OPENSHIFT_ALLOW_CLUSTER_READER,
-                DEFAULT_OPENSHIFT_ALLOW_CLUSTER_READER);
-
-        try (OpenShiftClient osClient = new DefaultOpenShiftClient(builder.build())) {
-            ClusterRoleBinding bResponse = osClient.clusterRoleBindings().inNamespace("").withName("cluster-admins")
-                    .get();
-
-            logger.debug("Does '{}' exist in cluster-admins? '{}'", username,
-                    bResponse.getUserNames().contains(username));
-
-            if (clusterReaderAllowed) {
-                ClusterRoleBinding readerResponse = osClient.clusterRoleBindings().inNamespace("")
-                        .withName("cluster-readers").get();
-
-                logger.debug("Does '{}' exist in cluster-readers? '{}'", username,
-                        readerResponse.getUserNames().contains(username));
-
-                if (readerResponse.getUserNames().contains(username))
-                    return true;
-            }
-
-            return bResponse.getUserNames().contains(username);
+    private boolean isOperationsUser(final String user, final String token) {
+        ConfigBuilder builder = new ConfigBuilder().withOauthToken(token);
+        boolean allowed = false;
+        try (NamespacedOpenShiftClient osClient = new DefaultOpenShiftClient(builder.build())) {
+            logger.debug("Submitting a SAR to see if '{}' is able to retrieve logs across the cluster", user);
+            SubjectAccessReviewResponse response = osClient.inAnyNamespace()
+                    .subjectAccessReviews()
+                    .createNew()
+                    .withVerb("get")
+                    .withResource("pod/logs")
```

Comment:
Correct, but @deads2k's comments about `pods/log` and `/apis/authorization.k8s.io/v1/selfsubjectaccessreviews` still stand.  The former should definitely be addressed, and the latter is important since it does not make sense for new code to use a legacy endpoint.